http/src/main/java/com/redhat/lightblue/client/http/transport/JavaNetHttpTransport.java

```diff
@@ -126,8 +149,6 @@ public String executeRequest(LightblueRequest request, String baseUri) throws Li
             }
 
             return response(connection);
-        } catch (ProtocolException e) {
-            throw new LightblueHttpClientException(e);
```

Comment:
Since ProtocolException is an IOException, and they both do the same thing, I thought it was cleaner / DRY to remove this block.